modules/462-loki/templates/statefulset.yaml

```diff
@@ -37,6 +40,44 @@ spec:
       {{- include "helm_lib_priority_class" (tuple . "cluster-low") | nindent 6 }}
       {{- include "helm_lib_module_pod_security_context_run_as_user_nobody_with_writable_fs" . | nindent 6 }}
       containers:
+      - name: kube-rbac-proxy
+        {{- include "helm_lib_module_container_security_context_read_only_root_filesystem" . | nindent 8 }}
+        image: {{ include "helm_lib_module_common_image" (list . "kubeRbacProxy") }}
+        args:
+          - "--secure-listen-address=$(KUBE_RBAC_PROXY_LISTEN_ADDRESS):3100"
+          - "--v=2"
+          - "--logtostderr=true"
+          - "--stale-cache-interval=1h30m"
+        ports:
+          - containerPort: 3100
+            name: loki
+        env:
+          - name: KUBE_RBAC_PROXY_LISTEN_ADDRESS
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: KUBE_RBAC_PROXY_CONFIG
+            value: |
+              upstreams:
+              - upstream: http://127.0.0.1:3099/
+                path: /
+                authorization:
+                  resourceAttributes:
+                    namespace: d8-monitoring
+                    apiGroup: apps
+                    apiVersion: v1
+                    resource: statefulsets
+                    subresource: logs
+                    name: loki
```

Comment:
You also need access rights to Prometheus to scrape this endpoint, like here:
https://github.com/deckhouse/deckhouse/blob/main/modules/200-operator-prometheus/templates/rbac-to-us.yaml

Besides, it is also required to give access to grafana and log-shipper to loki:
1. A token should be added here https://github.com/deckhouse/deckhouse/blob/c1ff4bf5f1b241af666f1dfe2d27947e4e694f93/modules/462-loki/templates/logshipper-log-destination.yaml#L7-L10
2. And here https://github.com/deckhouse/deckhouse/blob/c1ff4bf5f1b241af666f1dfe2d27947e4e694f93/modules/462-loki/templates/grafana-datasource.yaml#L8-L15